contrib/pg_tde/src/access/pg_tde_tdemap.c

```diff
@@ -1605,13 +1605,12 @@ pg_tde_fetch_wal_keys(XLogRecPtr start_lsn)
 					*return_wal_rec = NULL;
 	bool		new_file;
 
-
 	LWLockAcquire(lock_pk, LW_SHARED);
 	principal_key = GetPrincipalKey(rlocator.dbOid, LW_SHARED);
 	if (principal_key == NULL)
 	{
 		LWLockRelease(lock_pk);
-		elog(DEBUG1, "tde WAL read: no principal key");
+		elog(INFO, "fetch WAL keys: no principal key");
```

Comment:
debug leftover?

Reply:
Good catch, I forgot about that. We can't error here as that'll compromises all WAL reads if there is no wal principal key. But I'd still like to have some log traces. I changed DEBUG1=>INFO to see what's going on in pg_waldump. But it make sense reverse it back to DEBUG1 so it won't flood frontend output when no wal key is set (and the user might not have such an intention at all)